components/TopNav.vue

```diff
@@ -6,35 +6,36 @@
                     src="../assets/images/logo-32-px-color.svg" />Find a Doc
             </NuxtLink>
         </div>
-        <div>
+        <div >
             <NuxtLink to="/about">
                 About
             </NuxtLink>
-            <div class="cursor-pointer md:hidden">
-                <img src="../assets/images/hamburgerMenu.svg" alt="hamburger menu" />
+            <div  class="cursor-pointer md:hidden " >
+                <img @click="toggleMenu" src="../assets/images/hamburgerMenu.svg" alt="hamburger menu" />
+                <div v-if="showMenu">
```

Comment:
for this PR, this approach is fine. You should consider how you plan to show these same links in within the hamburger menu in the future work.